docs/source/schema/creating-directives.mdx

```diff
@@ -19,8 +19,11 @@ type Query {
 
```

Comment:
This is yet another file documenting somebody else's software. (In this case, software that used to be part of AS, then got factored out into a separate library, then the library got passed off to other people, and finally we removed the default integration.) I personally wish we didn't have so much of that, or that it was somewhere separate from the core AS docs. Maybe we consider leaving this file out for v4 and if there's popular demand we resurrect it?

Reply:
For this article, what if we left it out of the sidebar for now? So we'd still have the file on hand in case people want it, and folks searching for "creating directives" would still find something for their search, but the article is altogether less visible?

Reply:
That seems a bit confusing to me. Is this entirely about wanting to optimize SEO? Frankly I think people looking for documentation on this functionality of `@graphql-tools/schema` would be best served by finding the docs for `@graphql-tools/schema`.

Reply:
It's more like I already put in the effort to update this article and the corresponding example, so the lift has _already_ been done on our side of things. So if that can help people searching in our documentation, that'd be nice? I can remove it if you feel like it doesn't add value though!

I could alternatively remove this full article, but add a plug for the `docs-example` in the main Directives article for those interested? 

I don't have strong feelings here, so I'm down to do whatever you think is best!